### PR TITLE
Re-add changes to local assembly that got lost

### PR DESCRIPTION
**Pull request by mbhall88**
@rmcolq and @rffrancon can you please go through and make sure this is up-to-date with all your changes to.

### CODE REVIEW

**Comment by rmcolq** [comment]
I haven't edited those files, so changes all look good to me.

File: src/denovo_discovery/local_assembly.cpp [hunk 1]
```diff
@@ -109,28 +125,17 @@ void get_paths_between_util(const std::string &start_kmer, const std::string &en
...
-        // prevent the number of paths going above a fixed number
-        if (full_paths.size() > g_max_num_paths) {
-            BOOST_LOG_TRIVIAL(debug) << "Too many paths found " << std::to_string(full_paths.size())
-                                     << ". Skipping slice.";
-            full_paths.clear();
-            return;
-        }
     }
 
     for (unsigned int i = 0; i < num_children; ++i) {
```
Comment (rffrancon) [inline src/denovo_discovery/local_assembly.cpp]: `for (const auto &child_node: child_nodes) {`
no need for `num_children` variable, right?

File: src/denovo_discovery/local_assembly.cpp [hunk 2]
```diff
@@ -109,28 +125,17 @@ void get_paths_between_util(const std::string &start_kmer, const std::string &en
     auto &child_nodes = tree[start_kmer];
```
Comment (rffrancon) [inline src/denovo_discovery/local_assembly.cpp]: perhaps move this alias closer to where it's used?: after the if statement below

**Comment by rffrancon** [comment]
These changes don't touch any code that I've changed from what I can tell.

May I suggest squashing the commits.

**Comment by mbhall88** [comment]
> These changes don't touch any code that I've changed from what I can tell.
> 
> May I suggest squashing the commits.

When I look in my git log there is other commits from you in between some of these commits so I don't see a straightforward way of squashing them - well at least I don't trust myself to do it.

**Comment by mbhall88** [comment]
Ah cool, GitHub gave me the option to squash and merge them.

**Pull request merged**